docs/software/devtools/index.md

```diff
@@ -28,6 +28,6 @@ This ensures that computational resources are utilized to their fullest potentia
 Learning to analyze the performance of an applications effectively is crucial to build a deeper understanding of how your code interacts with the underlying hardware.
 In this section we introduce the various performance analysis solutions available at CSCS.
 
-* [Linaro Forge MAP][ref-devtools-map]
 * [NVIDIA Nsight Developer Tools][ref-devtools-nsight]
-
+* [Linaro Forge MAP][ref-devtools-map]
+* [VI-HPS Tools][ref-devtools-vihps]
```

Comment:
Don't forget to add the vihps.md to the `mkdocs.yaml` file so that it appears in the main index.